plugins/modules/pfsense_phpshell.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Martin Müller <martin.mueller@dataport.de>
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: pfsense_phpshell
+version_added: X
+author: Martin Müller
+short_description: PHP Shell
+description:
+  - Run a php shell
+options:
+  cmd:
+    description: PHP Code to run
+    required: true
+    type: str
+"""
+
+EXAMPLES = """
+- name: run phpshell with code pfSense config.xml
+  pfsense_phpshell:
+    cmd: |
+      require_once("filter.inc");
+      require_once("squid.inc");
+      squid_resync("yes");
+"""
+
+RETURN = """
+rc:
+  description: Status code after run php-shell (could be changed using `exit(x)`)
+  returned: always
+  type: int
+  sample:
+    - 0
+stdout:
+  description: Output of the php-shell (include your code)
+  returned: always
+  type: string
+  sample:
+    - |
```

Comment:
You say this will return a string but the sample is a YAML array (with one string element).